openmdao/docs/advanced_guide/derivs/partial_derivs_explicit.rst

```diff
@@ -0,0 +1,83 @@
+.. _advanced_guide_partial_derivs_explicit:
+
+OpenMDAO considers component derivatives to be **partial-derivatives**.
+The framework uses these partial-derivatives in order to compute the **total-derivatives** across your whole model.
+This tutorial is focused on how to define the partial-derivatives* for instances of :ref:`ExplicitComponent <comp-type-2-explicitcomp>`.
+
+
+
+****************************************************
+Defining Partial-Derivatives on Explicit Components
+****************************************************
+
+For any :ref:`ExplicitComponent <comp-type-2-explicitcomp>` instance you are going to provide derivatives of the **outputs with respect to the inputs**.
```

Comment:
instance is not the right term here..  these are things you do in a class definition.. I think you could just remove the word instance